scipy/_lib/doccer.py

```diff
@@ -80,14 +95,14 @@ def inherit_docstring_from(cls):
 
     Parameters
     ----------
-    cls : Python class or instance
+    cls : type | object
```

Comment:
I think we typically write `or` instead of `|` - at least that it was appears in [the style guide](https://numpydoc.readthedocs.io/en/latest/format.html#parameters)

Reply:
Ah I wasn't aware. In that case, you should probably use `or` 🤷🏻

Reply:
Should be fixed now.